modules/pubxaiAnalyticsAdapter.js

```diff
@@ -156,8 +169,12 @@ function send(data, status) {
         prebidVersion: $$PREBID_GLOBAL$$.version
       }
     });
-
-    ajax(pubxaiAnalyticsRequestUrl, undefined, JSON.stringify(data), { method: 'POST', contentType: 'text/plain' });
+    if (status == 'bidwon') {
+      ajax(pubxaiAnalyticsRequestUrl, undefined, JSON.stringify(data), { method: 'POST', contentType: 'text/json' });
+    } else if (status == 'auctionEnd' && !auctionCache.includes(data.initOptions.auctionId)) {
```

Comment:
@pnhegde I found the issue!

IE does not support `Array.includes`!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Please change to:

```js
auctionCache.indexOf(data.initOptions.auctionId) === -1
```

Reply:
@robertrmartinez Thanks a lot for your inputs. I have made suggested changes and it seem to be working fine.